Makefile

```diff
@@ -1,19 +1,10 @@
 all:
 	export TIMED
-	$(MAKE) -C theories all
+	@+$(MAKE) -C theories all
 
-install:
-	$(MAKE) -C theories install
+force Makefile: ;
```

Comment:
What does this line do? I would have maybe expected `force: Makefile`

Reply:
To not pass down the "Makefile" target.